libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/metastore/DomainMetaStore.java

```diff
@@ -45,6 +47,13 @@
      */
     void setBusinessServiceDomain(final String domainName, final String businessService);
 
+    /**
+     * Get a list of valid business services
+     * @param userName (optional) if not null, only get business services associated with the user
+     * @return Business Services List
+     */
+    List<String> getValidBusinessServices(final String userName);
```

Comment:
we should also add similar api methods for the other attributes that the meta store supports:

getValidAWSAccounts()
getValidAzureSubscriptions(),e tc.